app/models/local_authority.rb

```diff
@@ -15,4 +15,31 @@ class LocalAuthority < ResponsibleBody
   }
 
   validates :organisation_type, presence: true
+
+  def la_funded_place
+    schools.la_funded_place_establishment_type&.first
```

Comment:
If there is nothing in the array and this returns a `nil` does this cause you a problem? Is there a spec for that case?

Reply:
tonyheadford 1 hour ago Contributor
The & is superfluous here

Suggested change 
    schools.la_funded_place_establishment_type&.first
    schools.la_funded_place_establishment_type.first